src/clients/AcrossAPIClient.ts

```diff
@@ -1,6 +1,6 @@
 import { winston, BigNumber, getL2TokenAddresses } from "../utils";
 import axios, { AxiosError } from "axios";
-import { HubPoolClient } from "./HubPoolClient";
+import { HubPoolClient } from ".";
```

Comment:
I recall tsc complaining at me for using `"."` instead of `"./"`.

```suggestion
import { HubPoolClient } from "./";
```

Reply:
This is now removed